### ISSUE TITLE
BUG: Keep command ignored depending on spaces around +

### ISSUE THREAD

**Issue by AllieWade** [Humblemonk/DiceMaiden#175]
**Describe the bug**
When using the keep command, it is parsed differently depending on the spaces around a modifier on the roll.

**To Reproduce**
Test the following commands, paying attention to the inclusion or exclusion of spaces.
```!roll 5d6 + 2 k4```
(spaces before and after +)
Parses correctly: the highest 4 dice are kept and totaled, and then 2 is added to the total.

```!roll 5d6+2 k4```
(no spaces around +)
Parses incorrectly: all 5 dice are totaled, and 2 is added. The k4, like goggles, does nothing.

```!roll 5d6 +2 k4```
(space before +)
Parses incorrectly: all 5 dice are totaled, and 2 is added.

```!roll 5d6+ 2 k4```
(space after +)
Parses correctly: highest 4 dice are totaled, and 2 is added.

In other words, it appears that the space *after* the "+" is the important one.

**Expected behavior**
I would expect all four of these commands to perform identically, totaling the highest 4 and adding 2.

**Comment by Humblemonk** [comment]
Hey! So I am having trouble reproducing this bug. 

I attempted all of the rolls you outlined and the results look to be accurate:

```[10:48 PM] Humblemonk: !r 5d6 + 2 k4BOT
 Dice Maiden: Humblemonk Roll: [4, 4, 3, 2, 1] Result: 15 Request: [5d6 + 2 k4]

 Humblemonk: !r 5d6+2 k4
 Dice Maiden: Humblemonk Roll: [6, 5, 1, 1, 1] Result: 15 Request: [5d6+2 k4]

Humblemonk: !r 5d6 +2 k4
Dice Maiden: Humblemonk Roll: [5, 4, 2, 1, 1] Result: 14 Request: [5d6 +2 k4]

Humblemonk: !r 5d6+ 2 k4
Dice Maiden: Humblemonk Roll: [6, 6, 6, 5, 4] Result: 25 Request: [5d6+ 2 k4]

**Comment by AllieWade** [comment]
Okay, it turns out I'm a doofus. I went back and checked my original rolls and while I thought I had tested with only the k4 operator, I was wrong. The spaces aren't the problem at all: the problem is combining the k operator with the s operator, which I later realized wasn't even necessary:

```Sylveon of the Floating Throne — Today at 10:07 PM
!roll 5d6 + 2 k4 s
Dice Maiden
BOT
 — Today at 10:07 PM
Sylveon of the Floating Throne Roll: [6, 6, 6, 4, 2] Result: 26```

And that happens no matter what you do with the spaces. I'm so sorry for misreporting this.
